components/Tabs/Tabs.js

```diff
@@ -1,72 +1,70 @@
-    // this.cards = Array.from(this.cards).map();
+     this.cards = Array.from(this.cardElements).map(cardElement => new TabCard(cardElement));
 
-    // Add a click event that invokes this.selectTab
-    // this.tabElement.addEventListener();
-  }
+     // Add a click event that invokes this.selectTab
+     this.tabElement.addEventListener('click', () => this.selectTab());
+   }
 
-  selectTab(){
+   selectTab() {
 
-    // Select all elements with the .tab class on them
-    // const tabs = document.querySelectorAll();
-    
-    // Iterate through the NodeList removing the .active-tab class from each element
-    // tabs.forEach()
+     // Select all elements with the .tab class on them
+     const tabElements = document.querySelectorAll(".tab");
 
-    // Select all of the elements with the .card class on them
-    // const cards = ;
+     // Iterate through the NodeList removing the .active-tab class from each element
+     tabElements.forEach(tabElement => tabElement.classList.remove('active-tab'));
 
-    // Iterate through the NodeList setting the display style each one to 'none'
-    // cards.forEach()
-    
-    // Add a class of ".active-tab" to this.tabElement
-    // this.tabElement;
-  
-    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class. Just un-comment the code and study what is happening here.
-    // this.cards.forEach(card => card.selectCard());
-  }
-}
+     // Select all of the elements with the .card class on them
+     const cardElements = document.querySelectorAll(".card");;
+
+     // Iterate through the NodeList setting the display style each one to 'none'
+     cardElements.forEach(cardElement => cardElement.style.display = 'none');
 
-class TabCard {
-  constructor(cardElement){
-    // Assign this.cardElement to the cardElement DOM reference
-    // this.cardElement;
-  }
-  selectCard(){
-    // Update the style of this.cardElement to display = "flex"
-    // this.cardElement;
-  }
+     // Add a class of ".active-tab" to this.tabElement
+     this.tabElement.classList.add('active-tab');
+     // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class. Just un-comment the code and study what is happening here.
+     this.cards.forEach(card => card.selectCard());
+   }
+ }
 
-}
+ class TabCard {
+   constructor(cardElement) {
+     // Assign this.cardElement to the cardElement DOM reference
+     this.cardElement = cardElement;
+   }
+   selectCard() {
+     // Update the style of this.cardElement to display = "flex"
+     this.cardElement.style.display = "flex";
+   }
 
-/* START HERE: 
+ }
 
-- Select all classes named ".tab" and assign that value to the tabs variable
+ /* START HERE: 
+  
+ - Select all classes named ".tab" and assign that value to the tabs variable
```

Comment:
You're code is working so I'm not sure  if you need to do this, but I think this line means that you should set the variable tabs equal to  document.querySelectorAll(".tab").forEach(tab => new TabLink(tab)). 

(i.e.  let tabs =  document.querySelectorAll(".tab").forEach(tab => new TabLink(tab)) ;)